static/src/deploys-radiator/app/api.ts

```diff
@@ -43,15 +43,15 @@ export const getDifference = (base: string, head: string): Promise<List<GitHubCo
                         {
                             url: gitHubCommitJson.html_url,
                             authorName: gitHubCommitJson.commit.author.name,
-                            authorLogin: gitHubCommitJson.author.login,
+                            authorLogin: (gitHubCommitJson.author) ? gitHubCommitJson.author.login : 'Unknown',
```

Comment:
Thanks for the explanation. Looks like typescript 2 brings a lot of improvements. :)